### ISSUE TITLE
Version 1.2.0 raises TypeError  

### ISSUE THREAD

**Issue by Hoxbro** [iFaxity/wait-on-action#23]
I just tried to update to version 1.2.0 in our CI and got this error:

![image](https://github.com/iFaxity/wait-on-action/assets/19758978/bc85dedf-9445-4169-b068-7351b1fb0cb7)

This  works on  v1.1.0

``` yaml
  - name: Wait for JupyterLab
    uses: ifaxity/wait-on-action@v1.2.0
    with:
      resource: http-get://localhost:8887/lab
      timeout: 180000
```

The run: https://github.com/holoviz/panel/actions/runs/8811214270/job/24184864579?pr=6775

**Comment by grzanka** [comment]
I get the same error

**Comment by mlahargou** [comment]
Should [this ](https://github.com/iFaxity/wait-on-action/blob/df89d0cf8089bb0c38e25279c74848ef313da53b/src/main.ts#L1)be:

```ts
import * as core from '@actions/core';
```

**Comment by iFaxity** [comment]
@mlahargou great, I'll have a look at it and make a PR with the change. Thanks 😃.

**Comment by iFaxity** [comment]
A fix is now released as v1.2.1-rc.1. Could anyone here verify that it is working outside my testing? @mlahargou, @grzanka or @Hoxbro.

**Comment by grzanka** [comment]
How can be RC tested with GitHub actions ? Is it enough to change version in the yml file to `v1.2.1-rc.1` ?

**Comment by grzanka** [comment]
Seems to work with `v1.2.1-rc.1` and following changes:
https://github.com/yaptide/ui/pull/1606/files

<img width="1068" alt="Screenshot 2024-05-04 at 10 02 26" src="https://github.com/iFaxity/wait-on-action/assets/7374733/35581484-a2e9-41b7-ab63-fbd6e460e358">

**Comment by iFaxity** [comment]
@grzanka yes github actions uses tags from the actions github repo. 

Nice, i'll merge it to main and it will be available as both the v1 tag as well as v1.2.1 tag.

**Comment by iFaxity** [comment]
I've updated the v1 tag now so it points to the v1.2.1 release with the fix.
